### ISSUE TITLE
EnumNamespaces Rule Issue?

### ISSUE THREAD

**Issue by PompeiaPaetenari** [nicklockwood/SwiftFormat#811]
Second one I found... this rule seems to be hit-or-miss when I build the code after leaving it enabled, but seems another one we need to disable for our config.  Here is a simple example I encountered:

```swift
public class Resources {
  public static func getImage(named imageName: String) -> UIImage? {
    return UIImage(named: imageName, in: Bundle(for: self), compatibleWith: nil)
  }
}
```
Got converted to an `enum` and then caused the build to fail with: `Cannot convert value of type 'Resources.Type' to expected argument type 'AnyClass' (aka 'AnyObject.Type')` and has underlined `self`.

This might be easier to check (than my other issue) for with the syntax tree for this code block?

**Comment by nicklockwood** [comment]
Thanks for reporting this. Looks like an edge case that I should be able to fix.

**Comment by nicklockwood** [comment]
@PompeiaPaetenari fixed in 0.47.5.
